app/src/screenshotTest/kotlin/SearchScreenScreenshotTest.kt

```diff
@@ -0,0 +1,27 @@
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.developersbreach.composeactors.R
+import com.developersbreach.composeactors.data.datasource.fake.fakePersonsList
+import com.developersbreach.composeactors.ui.screens.search.ActorSearch
+import com.developersbreach.composeactors.ui.screens.search.SearchScreenUI
+import com.developersbreach.composeactors.ui.theme.ComposeActorsTheme
+
+class SearchScreenScreenshotTest {
+    @Preview(showBackground = true)
+    @Composable
+    private fun SearchScreenUILightPreview() {
+        ComposeActorsTheme(darkTheme = false) {
+            SearchScreenUI(
```

Comment:
Don't need to call the actual composable here, you can directly reuse the composable which is under preview.
I've applied that change in my commit.

Follow same approach in later tests too, so this whole class be replaced with calling previewable composable directly as below.

```kotlin
@Preview
@Composable
private fun SearchScreenUIPreviewTest() {
    SearchScreenUILightPreview()
}
```